### DISCUSSION TITLE
How to change datetime_string while using create_parent function

### DISCUSSION THREAD

**Discussion by harikiran2ndq** [pgpartman/pg_partman#380] [Q&A] [answered]
Hi,

First time using pg_partman, looking for help on changing datetime_string while using create_parent function on PG13.

My parent table DDL:

CREATE TABLE foo (
    id bigint NOT NULL,
    user_id bigint NOT NULL,
    created_at timestamptz NOT NULL
)
PARTITION BY RANGE(created_at);

Using the following create_parent function call:

SELECT partman.create_parent(p_parent_table => 'public.foo',
 p_control => 'created_at',
 p_type => 'native',
 p_interval=> 'monthly',
 p_start_partition=> '01-SEP-2021',
 p_premake => 2);

the following tables are created:
foo_default
foo_p2021_09
foo_p2021_10

the entry in table "part_config.datetime_string" column value is set to YYYY_MM

I'd like to create the partitions with datetime_string YYYYMM, meaning:
foo_p202109
foo_p202110

how do I change the naming convention? thanks in advance

**Comment by keithf4** [comment, marked as answer]
So I'd originally designed pg_partman without intending to allow custom datetime_strings since it was an easier design. Hadn't had the need or request to allow that since then and it's worked out well, so I hadn't worked on purposely allowing that. It may just work changing the string value and renaming your tables afterwards, but I cannot guarantee that.
